src/routes/v1/socialRoutes.js

```diff
@@ -0,0 +1,28 @@
+import {
+  Router
+} from 'express';
+import passport from 'passport';
+import SocialController from '../../controllers/socialController';
+
+const router = Router();
```

Comment:
We are not using this convention. Please look at other route files and check how we implement routes.